### PR TITLE
aws-secretsmanager: update page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/aws-secretsmanager.md [hunk 1]
```diff
@@ -7,11 +7,15 @@
...
+
 - Create a secret:
 
-`aws secretsmanager create-secret --name {{name}} --description "{{secret_description}}" --secret-string {{secret}}`
+`aws secretsmanager create-secret --name {{name}} --description "{{secret_description}}" --secret-string '{{secret}}'`
 
-- Delete a secret:
+- Delete a secret (append `--force-delete-without-recovery` to delete immediately without any recovery period):
 
 `aws secretsmanager delete-secret --secret-id {{name_or_arn}}`
```
Comment (spageektti) [inline pages/common/aws-secretsmanager.md:20]:
```suggestion
`aws secretsmanager delete-secret --secret-id {{name|arn}}`
```

File: pages/common/aws-secretsmanager.md [hunk 2]
```diff
@@ -7,13 +7,17 @@
...
 - Create a secret:
 
-`aws secretsmanager create-secret --name {{name}} --description "{{secret_description}}" --secret-string {{secret}}`
+`aws secretsmanager create-secret --name {{name}} --description "{{secret_description}}" --secret-string '{{secret}}'`
 
-- Delete a secret:
+- Delete a secret (append `--force-delete-without-recovery` to delete immediately without any recovery period):
 
-`aws secretsmanager delete-secret --secret-id {{name_or_arn}}`
+`aws secretsmanager delete-secret --secret-id {{name|arn}}`
```
Comment (spageektti) [inline pages/common/aws-secretsmanager.md:20]:
```suggestion
`aws secretsmanager delete-secret --secret-id {{name|arn}} {{--force-delete-without-recovery}}`
```

Comment (fazlearefin) [reply pages/common/aws-secretsmanager.md:20]: I don't think we need this change as the example description already says, inside brackets, to as this option if the user doesn't want recovery.

Comment (Waples) [reply pages/common/aws-secretsmanager.md:20]: As @fazlearefin already described, I don't think this change is needed. Can be closed tbh

Comment (spageektti) [reply pages/common/aws-secretsmanager.md:20]: > I don't think we need this change as the example description already says, inside brackets, to add this option if the user doesn't want recovery. 

I know but I seen similar example in other page, and the optional flag was written in the placeholder syntax too

**Review by spageektti — COMMENTED** [review]
Other examples on this page uses `name_or_arn` instead of `name|arn`. This should be changed.

File: pages/common/aws-secretsmanager.md [hunk 3]
```diff
@@ -7,13 +7,17 @@
 
 `aws secretsmanager list-secrets`
 
+- List all secrets but only show the secret names and ARNs (easy to view):
+
+`aws secretsmanager list-secrets --query 'SecretList[*].{Name: Name, ARN: ARN}'`
```
[the remark this answers is not in the document]
Comment (fazlearefin) [reply pages/common/aws-secretsmanager.md:12]: Nope, this will not work. The string has to be used as it is.

**Comment by fazlearefin** [comment]
> Other examples on this page uses `name_or_arn` instead of `name|arn`. This should be changed.

Done.

**Review by spageektti — APPROVED** [review]

**Pull request merged**